spec/lib/statistics_spec.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Statistics do
+
+  def mom_changed_cases_growth(start_cases:, end_cases:)
+    Statistics.mom_changed_cases_growth(cases_at_start: start_cases, cases_at_end: end_cases)
+  end
+
+  def mom_cases_growth(start_cases:, end_cases:)
+    Statistics.mom_cases_growth(cases_at_start: start_cases, cases_at_end: end_cases)

```

Comment:
What happens if an invalid date or other data types are sent as arguments?

Reply:
Yeah, I have that as one of my things to tackle as part of this.  It's technically always been a concern, but everything was so hard coded that we didn't have to care.